changes/43.feature.rst

```diff
@@ -0,0 +1,2 @@
+Include zhmccli and zhmcclient versions as meta information
```

Comment:
The change fragment file needs to be named after the issue it resolves. Please either open an issue and change the filename accordingly, or use `noissue.2.feature.rst`.